certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/event/model/CertificateEventType.java

```diff
@@ -1,14 +1,16 @@
 package se.inera.intyg.certificateservice.domain.event.model;
 
 public enum CertificateEventType {
+
   READ("read-certificate", Constants.ACTION_TYPE_ACCESSED, null),
   CREATED("created-certificate", Constants.ACTION_TYPE_CREATION, null),
   UPDATED("updated-certificate", Constants.ACTION_TYPE_CHANGE, null),
   DELETED("deleted-certificate", Constants.ACTION_TYPE_DELETION, null),
   VALIDATED("validated-certificate", Constants.ACTION_TYPE_ACCESSED, null),
   SIGNED("sign-certificate", Constants.ACTION_TYPE_CHANGE, "certificate-sign"),
```

Comment:
Could you change the messageType to certificate-signed? Was missed in an earlier pr.